### PR TITLE
Cellranger: Count submodule updated to parse Antibody Capture summary

### PR DESCRIPTION
**Pull request by achakkas**
Updated the existing cellranger count module to show additional data from antibody tabs when present in the `web_summary.html`. 
+ Added `Antibody - Summary stats` for reports with antibody tabs
+ Added `Antibody - Counts Distribution Bargraph` for  reports with antibody tabs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@
 - Just run CI on the oldest + newest supported Python versions ([#2074](https://github.com/ewels/MultiQC/pull/2074))
 - Strict mode: rename `config.lint` to `config.strict`, crash early on module or template error. Add `MULTIQC_STRICT=1` ([#2101](https://github.com/ewels/MultiQC/pull/2101))
 - Trigger changelog entry addition on PR creation, in addition to an explicit comment to multiqc-bot ([#2102](https://github.com/ewels/MultiQC/pull/2102))
+- Added support for fature barcode data i.e Antibody (CITE-seq) in cellranger module ([#1909](https://github.com/ewels/MultiQC/issues/1909))
+- Cellranger count module updated to parse Gene Expression + Antibody Capture summary files ([#2118](https://github.com/ewels/MultiQC/pull/2118))
```
Comment (vladsavelyev) [inline CHANGELOG.md:15]: The bot added a separate changelog entry, so it's duplicated now. Note also that for module updates, it's nice to start the PR title with the module name followed by colon, so the bot puts the entry under the "Module updates" section:

### Module updates
- **Cellranger**: Count module updated to parse Gene Expression + Antibody Capture summary files ([#2118](https://github.com/ewels/MultiQC/pull/2118))

If you could remove the entries above and add one under "Module updates" as above, would be awesome!

Comment (vladsavelyev) [reply CHANGELOG.md]: I just pushed that into the branch :)

File: docs/images/screenshots/fastqc-run.svg
Comment (vladsavelyev) [inline docs/images/screenshots/fastqc-run.svg:1]: Why that was triggered 🤔

File: multiqc/modules/cellranger/count.py [hunk 2]
```diff
@@ -90,12 +108,22 @@ def parse_count_html(self):
                 )
 
             self.add_section(
-                name="Count - Summary stats",
+                name="Gene Expression - Summary stats",
```
Comment (vladsavelyev) [inline multiqc/modules/cellranger/count.py:111]: Why renaming only here, but not the other sections prefixed with "Count"?

Comment (vladsavelyev) [reply multiqc/modules/cellranger/count.py]: I'd keep the old name for now, if you don't mind

**Review by vladsavelyev — APPROVED** [review]
Thank you so much for the contribution!

**Comment by vladsavelyev** [comment]
@multiqc-bot changelog

File: CHANGELOG.md (hunk 1 above)
Comment (achakkas) [reply CHANGELOG.md]: Thanks! I'll keep this in mind for future PRs.

File: docs/images/screenshots/fastqc-run.svg
Comment (achakkas) [reply docs/images/screenshots/fastqc-run.svg:1]: That's my mistake, I manually ran an action which triggered this update.

File: multiqc/modules/cellranger/count.py (hunk 2 above)
Comment (achakkas) [reply multiqc/modules/cellranger/count.py]: Makes sense, thanks!

**Review by ewels — APPROVED** [review]
Great, would be nice to group together the Antibody report sections though, underneath the Count sections.

**Comment by chris-rands** [comment]
Thank you! Partially closes #1909

**Pull request merged**